.github/workflows/tests.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         env:
           TARANTOOL_SERVER_USER: root
           TARANTOOL_SERVER_GROUP: root
+          TARANTOOL_VERSION: "2.11.2-centos7"
```

Comment:
why didn't you delete this job? It's duplicate as a part of matrix tests

Reply:
I'v removed *TraantoolCintainer* classes from integration profile

Reply:
Got it. Then rename jobs, it's confusing because matrix tests are also integration tests

Reply:
tests-tarantool-container

tests-cartridge-container

tests-ee